pkg/images/integration/util.go

```diff
@@ -20,17 +21,18 @@ func GetMatchingImageIntegrations(registries []registryTypes.ImageRegistry,
 		}
 	}
 
+	ctx := context.Background()
```

Comment:
Might as well initialize this as a `var` on top instead of within each function call.

Reply:
changed it to pass parent context instead